tests/CMakeLists.txt

```diff
@@ -21,7 +21,11 @@ linalg_add_test(dot)
 linalg_add_test(gemm)
 linalg_add_test(gemv)
 linalg_add_test(givens)
-linalg_add_test(iterator)
+# (AMK 6/7/21) Temporarily removing iterator test since it doesn't build
+# [ 73%] Building CXX object tests/CMakeFiles/iterator.dir/iterator.cpp.o
+# /home/amklinv/stdBLAS/tests/iterator.cpp:12:28: error: 'all' has not been declared in 'std::experimental'
```

Comment:
The `all` value is called `full_extent` now:

https://github.com/kokkos/mdspan/blob/stable/include/experimental/__p0009_bits/full_extent_t.hpp

We would have preferred `all`, but it would have been a bit of a bold name grab in the `std` namespace.